types/aws-lambda/trigger/lex-v2.d.ts

```diff
@@ -136,3 +133,31 @@ export interface LexV2ImageResponseCardButton {
     text: string;
     value: string;
 }
+
+export type LexV2Slots = Record<string, null | LexV2ScalarSlotValue | LexV2ListSlotValue>;
```

Comment:
Reminded me to look at this, Including `null` in record values is technically wrong and a breaking change too. In theory, users can opt into better types with [`noUncheckedIndexedAccess`](https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess), but that also applies to arrays, which is annoying.

Reply:
So... shrug? Probably shouldn't take the breaking change.

Reply:
`null` is the actual value coming back from AWS if the slot is listed in the intent but doesn't have a value yet. Example in the event I added to `lex-v2-tests.ts`. I'm a big fan of using `noUncheckedIndexedAccess` but I'm not sure it's applicable here. I apologize if I have misunderstood.

Reply:
Ah no problem here then!